### ISSUE TITLE
Sync parameters are saved in the synchronized data

### ISSUE THREAD

**Issue by surli** [johannesjo/super-productivity#1695]
When using the Sync feature with a local file, the sync path is actually shared in the synchronized data which can be a problem when using in different machines with different paths. 

### Your Environment

- Version used: 7.8.0 - AUR version
- Operating System and version: Archlinux
- Desktop Environment: Cinnamon

### Expected Behavior

The sync path is not shared in the synchronized data, so that we can use different paths and keep data synchronized.

### Current Behavior

The sync path is synchronized.

### Steps to Reproduce (for bugs)

0. Assuming there's a nextcloud setup (but would work with any sync of file)
1. On computer 1, configure Superproductivity to sync on local file in path `/home/user/nextcloud/superproductivity`
2. Sync nextcloud on computer 2 with path for directory being: `/home/user2/nextcloud`
3. Configure superproductivity on computer 2 to Sync using `/home/user2/nextcloud/superproductivity`
4. When asking which data to use, answer to use remote data
5. See that the Sync path changed in preferences

**Comment by johannesjo** [comment]
Thanks for opening this up! This was a bug and should be fixed with the next release (or with the current snap edge release). Please let me know, if it works out for you.

**Comment by johannesjo** [comment]
@TimMoser92 they're not actively deleted. Does it work, if you remove them manually?

**Comment by TimMoser92** [comment]
I tried removing them manually in the synced file - they were readded with the next sync. However I should add that I don't use the local file sync anymore but switched to WebDAV (with Nextcloud). It still adds the setting to the synced file (along with all the other sync settings - especially the login data for WebDAV, but that is another issue).

I'm not sure, if it is relevant, but I also used the same cloud-file with the import function to initially setup SP on the second machine. I do not know if the imported file is used as data store afterwards (I hope not).
